packages/adblocker-electron/content.ts

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import {
+  IBackgroundCallback,
+  IMessageFromBackground,
+} from '@cliqz/adblocker-webextension-cosmetics';
+import { ipcRenderer, webFrame } from 'electron';
+
+import { extractFeaturesFromDOM } from '@cliqz/adblocker';
+
+const enableMutationObserver = true; // Not sure what to do with it.
```

Comment:
Does not have to be now, but this option of enabling mutation observer could be communicated from main process, so that we can configure it when creating the instance of `ElectronBlocker`.